client/components/index.js

```diff
@@ -11,6 +11,7 @@ export { default as AnimationSlider } from './animation-slider';
 export { default as Card } from './card';
 export { default as Chart } from './chart';
 export { default as ChartLegend } from './chart/legend';
+export { default as ChartPlaceholder } from './chart/placeholder';
```

Comment:
I was trying to stay consistent with what we did for the `Table` component, and now `SummaryList`. We could add an additional prop instead to the `Chart` component but I think this makes sense too.

Reply:
For `Table` & `SummaryList` it makes sense to have them separate, since we want to be able to pass through `numberOf___`. I agree that for consistency, we should keep `ChartPlaceholder` separate, too.